### ISSUE TITLE
Activity Panels: Visual Issues on Narrow Viewport in new navigation experience

### ISSUE THREAD

**Issue by louwie17** [woocommerce/woocommerce-admin#5571]
**Describe the bug**
When using the new navigation experience on a narrow viewport (main branch) a user can't open the activity panels until the main left nav is opened.
When the activity panel is open the items are being rendered off of the screen, it appears that the activity panel uses the viewport width, but it can't use the full-width given the 'W' logo is rendered on the left.
It doesn't show a close icon for the activity panel either.  Changes might be related to #5556 

**To Reproduce**
Using the main branch

1. Enable the new WooCommerce navigation experience
2. Open the WooCommerce home screen on a mobile device or change viewport in browser
3. Open the left nav (by clicking on the Wordpress logo)
4. Click on the Woo logo in top right

**Screenshots**

<img width="410" alt="Screen Shot 2020-11-09 at 2 40 15 PM" src="https://user-images.githubusercontent.com/2240960/98586410-09883d00-229f-11eb-99ab-9ee05066930d.png">
<img width="406" alt="Screen Shot 2020-11-09 at 2 28 57 PM" src="https://user-images.githubusercontent.com/2240960/98586415-0a20d380-229f-11eb-9247-f11035094259.png">

**Comment by timmyc** [comment]
As I noted in #5556 it looks like this is current behavior in 4.6.2 - so while it would be really nice to get this fixed for 1.7 final, since it isn't a new regression I'm giving it a low priority.

**Comment by chihsuan** [comment]
Looks like the layout is changed, no the woo logo and 'stock' icon. I'm closing this since this is over 6 months old.

![Screen Shot 2022-01-13 at 15 58 17](https://user-images.githubusercontent.com/4344253/149289096-270b0237-fd3e-451f-a8b2-3b573760fea2.png)
